docs/website/docs/dlt-ecosystem/verified-sources/rest_api.md

```diff
@@ -560,49 +560,81 @@ This will include the `id`, `title`, and `created_at` fields from the `issues` r
+        {"id": 3, "title": "Post 3", "created_at": "2024-01-28"}
+    ]
+}
+```
+
+To enable the incremental loading for this endpoint, you can use the following configuration:
+
+```py
+{
+    "path": "posts",
+    "data_selector": "results",  # Optional JSONPath to select the list of posts
+    "params": {
+        "created_since": {
             "type": "incremental",
-            "cursor_path": "updated_at",
-            "initial_value": "2024-01-25T11:21:28Z",
+            "cursor_path": "created_at", # The JSONPath to the field we want to track in each post
+            "initial_value": "2024-01-25",
         },
-    }
-    ```
+    },
+}
+```
 
-    This configuration tells the source to create an incremental object that will keep track of the `updated_at` field in the response and use it as a value for the `since` parameter in subsequent requests.
+After you run the pipeline, dlt will keep track of the last `created_at` from all the posts fetched and use it as the `created_since` parameter in the next request.
+So in our case, the next request will be made to `https://api.example.com/posts?created_since=2024-01-28` to fetch only the new posts created after `2024-01-28`.
 
-2. Specifying the `incremental` field in the [endpoint configuration](#endpoint-configuration):
+Now, let's break down the configuration. The `created_since` parameter is defined as an incremental parameter with the following fields:
 
-    ```py
-    {
-        "incremental": {
-            "start_param": "<parameter_name>",
-            "end_param": "<parameter_name>",
-            "cursor_path": "<path_to_cursor_field>",
-            "initial_value": "<initial_value>",
-            "end_value": "<end_value>",
-        }
-    }
-    ```
+```py
+{
+    "<parameter_name>": {
+        "type": "incremental",
+        "cursor_path": "<path_to_cursor_field>",
+        "initial_value": "<initial_value>",
+    },
+}
+```
+
+- `type`: The type of the incremental parameter. Set to `incremental`.
+- `cursor_path`: The JSONPath to the field within each item in the list that will be used as the cursor value. In this case, it's `created_at`. Note that the path starts from the root of the item (dict) and not from the root of the response.
+- `initial_value`: The initial value for the cursor. This is the value that will initialize the state of incremental loading. In this case, it's `2024-01-25`.
+
+### Incremental loading using the `incremental` field
 
-    This configuration is more flexible and allows you to specify the start and end conditions for the incremental loading.
+The alternative method is to use the `incremental` field in the [endpoint configuration](#endpoint-configuration). This method is more flexible and allows you to specify the start and end conditions for the incremental loading:
+
+```py
+{
+    "incremental": {
+        "start_param": "<parameter_name>",
+        "end_param": "<parameter_name>",
+        "cursor_path": "<path_to_cursor_field>",
```

Comment:
could you note in comments what types and formats these parameters have, e.g.

```
    "incremental": {
        "cursor_path": "<path_to_cursor_field>", # "created_at"
```

for me it's not clear what I should put in `start_param` and `end_param`

Reply:
Thanks for the comment, added more details in 70e43afc94719ec428730961827205e3e547ed31